scripts/deploy-validator.sh

```diff
@@ -2,6 +2,9 @@
 
 set -e
 
+# Make sure we're at the source of the repo.
+cd "$(dirname "${BASH_SOURCE[0]}")/.."
```

Comment:
Nit: I think it would be better to use `cargo locate-project` and/or `git rev-parse --show-toplevel` (depending on what's needed for the particular script) rather than assume the script's location will remain constant.

Reply:
I'd say these two options have stronger assumptions (`cargo` and `git`).